### PR TITLE
Add rbmaradio.com connector

### PR DESCRIPTION
**Pull request by joe-niland**
This is my first attempt at a connector. It handles the player on https://www.rbmaradio.com

I think the playing detection still needs work, but not sure.

### CODE REVIEW

File: connectors/v2/rbmaradio.js [hunk 1]
```diff
@@ -34,14 +34,14 @@ Connector.getDuration = function () {
 	var $progressControl = $('.c-player__controls > .c-player__progress > progress');
 	var duration = '';
 	if ($progressControl) {
-		duration = $progressControl.attr('max'); 
+		duration = $progressControl.attr('max');
```
Comment (alexesprit) [inline connectors/v2/rbmaradio.js]: There's missing `parseInt` call at this line. The `getDuration` function should return numbers.

**Comment by alexesprit** [comment]
image

**Comment by joe-niland** [comment]
Thanks for the review @alexesprit !

The show I was testing with didn't have individual track listing. Have fixed this now.
Also removed the track art selector because they only have one image per Show.

**Comment by joe-niland** [comment]
Thank you @alexesprit

**Pull request merged**